### PR TITLE
Adding post install configuration to prep cluster for CI work

### PR DESCRIPTION
**Pull request by dry923**
I'm not sure where this should land yet but I want to put it somewhere for visibility. This script can be the baseline for taking a osde2e installed cluster and configure it to function in the PerfScale CI pipeline.

### CODE REVIEW

**Comment by dry923** [comment]
Updated. @morenod I think we need to look at getting this merged in as it will be needed for the pipeline jobs

File: post_install_node_config.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                       "aws ec2 authorize-security-group-ingress --group-id ITEM --protocol tcp --port 20000-20109 --cidr 0.0.0.0/0",
+                       "aws ec2 authorize-security-group-ingress --group-id ITEM --protocol udp --port 20000-20109 --cidr 0.0.0.0/0",
+                       "aws ec2 authorize-security-group-ingress --group-id ITEM --protocol tcp --port 32768-60999 --cidr 0.0.0.0/0",
+                       "aws ec2 authorize-security-group-ingress --group-id ITEM --protocol udp --port 32768-60999 --cidr 0.0.0.0/0"]
+    for item in sec_group_list:
+        if item == "":
+            return
+        else:
+            for cmnd in sec_group_cmds:
+                my_cmd = cmnd.replace('ITEM',item)
```
Comment (morenod) [inline post_install_node_config.py:50]: this is causing lint error, it seems that my_cmd is never used

Line below, the subprocess is using sec_grp_cmd, should it be my_cmd?

File: post_install_node_config.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    process = subprocess.Popen(sec_grp_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    stdout,stderr = process.communicate()
+    sec_group = stdout.decode("utf-8")
+
+    sec_group = sec_group.replace(' ','')
+    sec_group = sec_group.replace('"','')
+    sec_group = sec_group.replace('\n',' ')
+    sec_group_list = list(sec_group.split(" "))
+
+    sec_group_cmds = [ "aws ec2 authorize-security-group-ingress --group-id ITEM --protocol tcp --port 22 --cidr 0.0.0.0/0",
```
Comment (morenod) [inline post_install_node_config.py]: E201 whitespace after '['

File: post_install_node_config.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+# Make aws related config changes such as security group rules etc
+def _aws_config(nodes,clustername):
+    vpc_cmd = ["aws ec2 describe-instances --query 'Reservations[*].Instances[*].[InstanceId,Tags[?Key==`Name`].Value|[0],State.Name,PrivateIpAddress,PublicIpAddress, PrivateDnsName, VpcId]' --output text | column -t | grep " + clustername + "| awk '{print $7}' | grep -v '^$' | sort -u"]
+    process = subprocess.Popen(vpc_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    stdout,stderr = process.communicate()
+    cluster_vpc = stdout.decode("utf-8")
+    cluster_vpc = cluster_vpc.replace('\n','')
+    cluster_vpc = cluster_vpc.replace(' ','')
+
```
Comment (morenod) [inline post_install_node_config.py]: W293 blank line contains whitespace

File: post_install_node_config.py (hunk 1 above)
Comment (dry923) [reply post_install_node_config.py:50]: good catch resolved.

File: post_install_node_config.py (hunk 2 above)
Comment (dry923) [reply post_install_node_config.py]: fixed

File: post_install_node_config.py (hunk 3 above)
Comment (dry923) [reply post_install_node_config.py]: fixed

**Comment by dry923** [comment]
note do not merge yet. still testing

**Review by morenod — APPROVED** [review]
LGTM

**Pull request merged**